printer_mods/README.md

```diff
@@ -205,6 +205,7 @@ like so:
 | Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |:x:|
 | jatson | [Plug panel mod for Schurter CD44.1101.151 power entry](./jatson/plug_panel_schurter) | A modified plug panel for Voron 2.4r1 with Schurter CD44.1101.151 power entry modul. The panel has a LAN socket as well. | :x: | :grey_question: | :heavy_check_mark: | :x: |:x:|
 | jaysuk | [4.3 Fly Screen Mount](./jaysuk/4.3_fly_screen_mount) | A mount for a 4.3" Fly Screen | :heavy_check_mark: | :x: | :x: | :x: |:x:|
+| jbisson9 | [Backside Cable Management](./jbisson9/Backside_Cable_Management) | Routing wiring for the exhaust filter and BBT smart filament sensor | :x: | :grey_question: | :heavy_check_mark: | :x: | :grey_question:  |
```

Comment:
typo here, `BBT` should be `BTT`

Reply:
@MapleLeafMakers done

Reply:
@MapleLeafMakers fixed the stls and readme